src/ui/patch_hub/patch_hub_core.sh

```diff
@@ -159,6 +159,16 @@ function show_registered_mailing_lists()
   message_box='Below, you can see the lore.kernel.org mailing lists that you have registered.'$'\n'
   message_box+='Select a mailing list to see the latest patchsets sent to it.'
 
+  fallback_message='kw could not find any registered mailing list.'$'\n'$'\n'
```

Comment:
Don't forget to declare this variable at the top of the `show_registered_mailing_lists()` function (below `local message_box`)